views/client.ejs

```diff
@@ -31,11 +31,11 @@
       </div>
       <div class="form-group mx-sm-3 mb-2">
         <label for="port" class="sr-only">Port</label>
-        <input type="text" class="form-control port" id="port" placeholder="Port Number">
+        <input type="text" class="form-control port" id="port" placeholder="Port Number (Optional)">
```

Comment:
they should be at the `label`. Port number should be an example of what the user is supposed to put in.